doc/benchmarking/renate_benchmarks.rst

```diff
@@ -173,6 +180,14 @@ The following table contains the list of supported datasets.
       - multiple
       - Any `Hugging Face dataset <https://huggingface.co/datasets>`__ can be used. Just prepend the prefix ``hfd-``, e.g., ``hfd-rotten_tomatoes``. Select input and target columns via ``config_space``, e.g., add ``"input_column": "text", "target_column": "label"`` for the `rotten_tomatoes <https://huggingface.co/datasets/rotten_tomatoes>`__ example.
       - Please refer to `the official documentation <https://huggingface.co/datasets>`__.
+    * - CDDB
+      - Image Classification
+      - 2 classes from 5 domains of generated images using various techniques. Numbers vary across domains.
+      - Li, Chuqiao, et al. A continual deepfake detection benchmark: Dataset, methods, and essentials. IEEE/CVF Winter Conference on Applications of Computer Vision. 2023.
+    * - CORe50
+      - Object Recognition - image classfication
+      - 10 classes of 7 training domains, 3 other domains used for evaluation
```

Comment:
There are 8 domains. The 3 other domains is irrelevant for evaluation. Rather mention that there is only 1 test dataset which will be used as test set for each task. Same comment as before "10 classes of 7 training domains". What does that mean? Rather something like "50 classes, 8 domains (0-7)." Note, there are 50 classes, not 10.